include/dpp/permissions.h

```diff
@@ -127,6 +127,44 @@ class DPP_EXPORT permission {
 	 */
 	operator nlohmann::json() const;
 
+	/**
+	 * @brief Check for certain permissions, taking into account administrator privileges. It uses the Bitwise AND operator
+	 * @tparam T one or more uint64_t permission bits
+	 * @param values The permissions (from dpp::permissions) to check for
+	 *
+	 * **Example:**
+	 *
+	 * ```cpp
+	 * bool is_mod = permission.can(dpp::p_kick_members, dpp::p_ban_members);
+	 * // Returns true if it has permission to p_kick_members and p_ban_members
+	 * ```
+	 *
+	 * @return bool True if it has **all** the given permissions or dpp::p_administrator
+	 */
+	template <typename... T>
+	constexpr bool can(T... values) const noexcept {
+		return has(values...) || (value & p_administrator);
+	}
+
+	/**
+	 * @brief Check for certain permissions, taking into account administrator privileges. It uses the Bitwise AND operator
```

Comment:
It uses the Bitwise OR operator**

Reply:
huh. it uses the has method which uses the AND operator

Reply:
well i could remove the "It uses the Bitwise AND operator" entirely as this is irrelevant there

Reply:
Oh nvm yeah it uses AND after OR i got confused oop